internals/secrethub/org_init.go

```diff
@@ -39,14 +39,30 @@ func (cmd *OrgInitCommand) Register(r Registerer) {
 func (cmd *OrgInitCommand) Run() error {
 	var err error
 
+	client, err := cmd.newClient()
+	if err != nil {
+		return errio.Error(err)
+	}
+
+	user, err := client.Me().GetUser()
+	if user.Username == "Replace this predicate with a check whether user is on free plan" {
```

Comment:
What do we put here?

Reply:
The plan of the user is not relevant. For users, we offer only one plan: the free developer account.

Each organization has its own plan. So when the same user would create a second org, this would start again on a trial.

Therefore, no predicate is needed here, when creating a new organization, it always starts on a trial.